changelogs/fragments/ansible-test-python-3.12.yml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
+  - ansible-test - Add support for testing with Python 3.12.
+  - ansible-test - Update pip and setuptools versions.
```

Comment:
Can we specify the versions in the change log? I think, this information can be useful.

Reply:
added